db/migrate/20210119082215_add_nickname_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddNicknameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :nickname, :string, null: true, after: :email
```

Comment:
you should manage uniqueness through db too
So it should be uniq if value is not nil, and we shouldn't check uniqueness if value nil